module/Database/view/database/member/remove-address.phtml

```diff
@@ -7,8 +7,8 @@ $form->setAttribute(
     $this->url(
         'member/show/remove-address',
         [
-            'id' => $member->getMember()->getLidnr(),
-            'type' => $member->getType()->value,
```

Comment:
Funnily enough, I spent half an hour looking for a bug which already was in the GitHub review panel but was not altered. 
This bug made the form post to the wrong URL (e.g. containing `ordinary` instead of `student`)

Reply:
Hadn't even noticed that. Thanks for fixing it!